roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -38,7 +38,11 @@ server {
   include ssl.conf;
   include ssl-stapling.conf;
 
-  add_header Strict-Transport-Security "max-age=31536000; includeSubdomains; preload";
+  {% set hsts_max_age = item.value.ssl.hsts_max_age | default(nginx_hsts_max_age) %}
+  {% set hsts_include_subdomains = item.value.ssl.hsts_include_subdomains | default(nginx_hsts_include_subdomains) %}
+  {% set hsts_preload = item.value.ssl.hsts_preload | default(nginx_hsts_preload) %}
+
+  add_header Strict-Transport-Security "max-age={{ hsts_max_age }}{{ hsts_include_subdomains | ternary('; includeSubdomains', '') }}{{ hsts_preload | ternary('; preload', '') }}";
```

Comment:
do we care how messy this is? If the variables were strings it could be:

```
add_header Strict-Transport-Security "max-age={{ [hsts_max_age, hsts_include_subdomains, hsts_preload].join(';') }}";
```